Utils/ArrayHelper.php

```diff
@@ -52,4 +52,9 @@ public static function resolve(array $array, $key, $fallback = null)
 
         return $fallback;
     }
+
+    public static function get(array $array, $key, $fallback = null)
+    {
+        return array_key_exists($key, $array) ? $array[$key] : $fallback;
```

Comment:
Normally i'm in favor of strict checks like `array_key_exists` but not in that case - `array_key_exists` is a "little" bit slower compared to `isset` and this function will be called many times - so that micro optimization 'd give a lot. Here is some stackoverflow content about it: https://stackoverflow.com/a/9522522/3542877